src/components/vector/ExportDialog.tsx

```
@@ -46,321 +50,552 @@ export function ExportDialog({ open, onOpenChange }: ExportDialogProps) {
-          const dotRadius = settings.vectorLength * 0.15;
-          return `<circle id="${id}" cx="0" cy="0" r="${dotRadius}" fill="${settings.vectorColor}" ${transform} />`;
-        
+          // Punto
+          return `<circle cx="${actualLength / 2}" cy="0" r="${(settings.vectorWidth ?? 2) * 1.5}" fill="${settings.vectorColor ?? '#FFFFFF'}" ${transform}/>`;
+          
         case 'triangle':
           // Triángulo
-          const h = settings.vectorLength * 0.4;
-          return `<polygon id="${id}" points="0,-${h} ${h/2},${h/2} -${h/2},${h/2}" fill="none" ${stroke} ${transform} />`;
-        
+          const angle1 = Math.PI * 0.8; // 144 grados
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)